app/src/test/java/protect/card_locker/LoyaltyCardViewActivityTest.java

```diff
@@ -1389,4 +1391,51 @@ public void importCardOldFormat() {
         checkAllFields(activity, ViewMode.ADD_CARD, "Example Store", "", context.getString(R.string.anyDate), context.getString(R.string.never), "0", context.getString(R.string.points), "123456", context.getString(R.string.sameAsCardId), "Aztec", null, null);
         assertEquals(-416706, ((ColorDrawable) activity.findViewById(R.id.thumbnail).getBackground()).getColor());
     }
+
+
+    @Test
+    public void longPressOnBarcodeShouldCopyTheBarcodeValue() {
```

Comment:
This test name isn't completely accurate as it tests both the long and short click flows :)

```suggestion
    public void testCopyBarcodeValue() {
```